### YamlMime:JavaType
uid: "com.azure.analytics.synapse.artifacts.models.SparkServiceError"
fullName: "com.azure.analytics.synapse.artifacts.models.SparkServiceError"
name: "SparkServiceError"
nameWithType: "SparkServiceError"
summary: "The SparkServiceError model."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "java.lang.Object.clone()"
- "java.lang.Object.equals(java.lang.Object)"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.hashCode()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.toString()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class SparkServiceError"
constructors:
- "com.azure.analytics.synapse.artifacts.models.SparkServiceError.SparkServiceError()"
methods:
- "com.azure.analytics.synapse.artifacts.models.SparkServiceError.getErrorCode()"
- "com.azure.analytics.synapse.artifacts.models.SparkServiceError.getMessage()"
- "com.azure.analytics.synapse.artifacts.models.SparkServiceError.getSource()"
- "com.azure.analytics.synapse.artifacts.models.SparkServiceError.setErrorCode(java.lang.String)"
- "com.azure.analytics.synapse.artifacts.models.SparkServiceError.setMessage(java.lang.String)"
- "com.azure.analytics.synapse.artifacts.models.SparkServiceError.setSource(com.azure.analytics.synapse.artifacts.models.SparkErrorSource)"
type: "class"
metadata: {}
package: "com.azure.analytics.synapse.artifacts.models"
artifact: com.azure:azure-analytics-synapse-artifacts:1.0.0-beta.4
